src/main/webapp/app/entities/feedback-suggestion.model.ts

```diff
@@ -0,0 +1,28 @@
+export class TextFeedbackSuggestion {
+    constructor(
+        public id: number | undefined,
```

Comment:
suggestion: use number? instead of number | undefined

Reply:
Does not work :(

![screenshot-2023-11-03_002065](https://github.com/ls1intum/Artemis/assets/9006596/f07b9626-d8aa-4a5b-8f78-e1345658d469)